app/script/view_model/content/InputBarViewModel.js

```diff
@@ -133,7 +138,12 @@ z.viewModel.content.InputBarViewModel = class InputBarViewModel {
     });
 
     this.editedMention = ko.observable(undefined);
-    this.currentMentions = [];
+    this.currentMentions = ko.observableArray();
```

Comment:
I actually think you don't need the power of Observables here. Nothing seem to react to that array changes, you can go for a good ol' javascript Array